pkg/k8shandler/indexmanagement/index_management.go

```diff
@@ -0,0 +1,180 @@
+package indexmanagement
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+
+	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/pkg/logger"
+	esapi "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
+)
+
+const (
+	PolicyNameApp   = "app-policy"
+	PolicyNameInfra = "infra-policy"
+	PolicyNameAudit = "audit-policy"
+
+	MappingNameApp   = "app"
+	MappingNameInfra = "infra"
+	MappingNameAudit = "audit.infra"
+
+	PollInterval = "15m"
+
+	HotPhaseAgeAsPercentOfMaxAge = 5
+)
+
+var (
+	AliasesApp   = []string{"app, logs.app"}
```

Comment:
This is still broken. If you pull the change https://github.com/openshift/cluster-logging-operator/pull/368/files#diff-b4c07388d3271f4cc3b22294dedc13d5R28, I can drop #368 entirely.

Reply:
We'll follow this up with a pr to fix just the aliases